mapeditor/mapcontroller.cpp

```diff
@@ -323,6 +324,48 @@ void MapController::commitObjectErase(int level)
 	main->mapChanged();
 }
 
+void MapController::copyToClipboard(int level)
+{
+	_clipboard.clear();
+	_clipboardShiftIndex = 0;
+	auto selectedObjects = _scenes[level]->selectionObjectsView.getSelection();
+	for(auto * obj : selectedObjects)
```

Comment:
Can obj be null? If not, maybe use &

Reply:
Since it's container of pointers, auto & will be translated into auto &*. There is no way to store references in containers in C++ (there is reference_wrapper which has underlying pointer as well)